source/partials/_breadcrumb.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-breadcrumbs" data-click-events data-click-category="Header" data-click-action="Breadcrumb clicked">
+  <ol class="govuk-breadcrumbs__list">
+    <li class="govuk-breadcrumbs__list-item">
+      <%= link_to 'GOV.UK services', 'https://www.gov.uk/service-toolkit#gov-uk-services', class: 'govuk-breadcrumbs__link' %>
+    </li>
+    <li class="govuk-breadcrumbs__list-item">
+      <%= link_to 'GOV.UK&nbsp;Pay', 'index', class: 'govuk-breadcrumbs__link' %>
+    </li>
+    <li class="govuk-breadcrumbs__list-item">
+      <%= current_page.data.title %>
```

Comment:
Can we keep the link for the final element of the breadcrumb? [This is the accepted Design System pattern](https://design-system.service.gov.uk/components/breadcrumbs/).

Reply:
Sure, it'll just be an empty anchor link though